nnvm/tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -946,38 +1043,70 @@ def test_forward_tanh():
         tf.nn.tanh(in1)
         compare_tf_with_tvm(inp_array, 'Placeholder:0', 'Tanh:0')
 
+#######################################################################
+# Mean
+# ----
+def test_forward_mean():
+    def check_mean(ishape, **kwargs):
+        inp_array = np.random.uniform(size=ishape).astype(np.float32)
+        with tf.Graph().as_default():
+            in1 = tf.placeholder(shape=inp_array.shape, dtype=inp_array.dtype)
+            tf.keras.backend.mean(in1, **kwargs)
+            compare_tf_with_tvm(inp_array, 'Placeholder:0', 'Mean:0', no_gpu=True)
+
+    check_mean((10, 8, 16, 32))
+    check_mean((10, 8, 16, 32), axis=(2,3))
+    check_mean((10, 8, 16, 32), axis=(1,2), keepdims=True)
+
 #######################################################################
 # Main
 # ----
 if __name__ == '__main__':
+    # Transforms
     test_forward_transpose()
-    test_forward_convolution()
-    test_forward_pooling()
     test_forward_reshape()
     test_forward_squeeze()
+    test_forward_pack()
+    test_forward_resize_bilinear()
+    test_forward_pad()
+    test_forward_gather()
+    #test_forward_stridedslice()
+
+    # Activations
     test_forward_sigmoid()
+    test_forward_relu()
+    test_forward_leaky_relu()
+    test_forward_elu()
+    test_forward_selu()
+    test_forward_tanh()
+
+    # Reductions
     test_forward_argminmax()
     test_forward_reduce()
+    test_forward_mean()
+
+    # NN
+    test_forward_convolution()
+    test_forward_pooling()
     if tf.__version__ == '1.4.1':
         _test_forward_concat_v2()
+    test_forward_lrn()
+    test_forward_l2_normalize()
+
+    # General
     test_forward_multi_input()
-    test_forward_pack()
+    test_forward_variable()
+
+    # End to End
     test_forward_inception_v3()
     test_forward_inception_v1()
     test_forward_mobilenet()
-    test_forward_variable()
-    test_forward_resize_bilinear()
-    test_forward_pad()
-    #test_forward_lstm()
-    #test_forward_stridedslice()
-    test_forward_gather()
+    test_forward_resnetv2()
+
+    # NLP
```

Comment:
instead of NLP, use RNN
Can move `test_forward_ptb()` to   `# End to End` or `Models`

Reply:
Done